docs/plugin-core.md

```diff
@@ -755,6 +756,42 @@ Delegate processing to an external pipeline under specifics conditions
 
 Returns **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** 
 
+### throttle
+
+Take `Object` and return the same object
+
+```json
```

Comment:
If you don't want to use `"` in data examples, use `js` syntax (instead of `json`)...